dev/io.openliberty.grpc.1.0.internal.client.monitor/bnd.bnd

```diff
@@ -22,6 +22,7 @@ Bundle-Description: gRPC Client Monitor, version ${bVersion}
 # packages from Java when using Java 8 or earlier and from the new shipped bundles for Java 9 and later.
 Import-Package: \
   !*.internal.*, \
+  !com.google.errorprone.*,\
```

Comment:
Update copyright date range to include 2023

Reply:
This is a new package for this version of netty libs?

Reply:
Yes without this one gets:
```
[12/10/2023, 16:09:44:740 BST] 00000019 ogService-96-io.openliberty.grpc.1.0.internal.client.jakarta E CWWKE0702E: Could not resolve module: io.openliberty.grpc.1.0.internal.client.jakarta [96]
  Unresolved requirement: Import-Package: com.google.errorprone.annotations
```
The annotations are only used for Googles Error Prone scanning tools and are not loaded by
the class loader.

Reply:
Copyright updated